test/Redistribution.test.ts

```diff
@@ -688,6 +689,10 @@ describe('Redistribution', function () {
           r_node_1 = await ethers.getContract('Redistribution', node_1);
           r_node_2 = await ethers.getContract('Redistribution', node_2);
 
+          // Set price base
+          const priceBase = await priceOracle.priceBase();
+          priceBaseString = priceBase.toString();
```

Comment:
it is string by default, isn't it?

Reply:
It is not, it's an Object. BigNumber object, so it needs to be cast down to something but this was not needed to be string as we just use the number, so good call on this